components/build/tekton-chains/chains-certs.yaml

```diff
@@ -113,3 +113,37 @@ spec:
       terminationGracePeriodSeconds: 30
       serviceAccount: chains-secrets-admin
       serviceAccountName: chains-secrets-admin
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: tekton-chains-signing-secret
```

Comment:
being deployed into default namespace:
```
Error creating: pods "tekton-chains-signing-secret-" is forbidden: error looking up service account default/chains-secrets-admin: serviceaccount "chains-secrets-admin" not found
```

Reply:
I tested this with `kubectl apply` on an existing cluster so it passed. thanks for catching this.